blark/iec.lark

```diff
@@ -159,14 +160,15 @@ nanoseconds: FIXED_POINT "ns"i
            | INTEGER "ns"i
 
 // B.1.2.3.2
-_daytime: day_hour ":" day_minute ":" day_second
+_daytime: day_hour ":" day_minute [ ":" day_second ]
```

Comment:
Double-checked with TwinCAT and docs - this is correct 👍 

https://infosys.beckhoff.com/english.php?content=../content/1033/tcplccontrol/html/tcplcctrl_time_of_day.htm&id=

Reply:
Oh, but I think we're missing a test case in `test_transformer` - unless I just missed it - for the optional seconds.

Reply:
I think we've got it in this one:

![image](https://user-images.githubusercontent.com/33275230/170323307-0bb2c6ce-d4e1-422b-914d-d15f06e3497d.png)

Reply:
At first, though, I thought you were right! Thanks for checking! 😄

Reply:
Ah, that's fair! I was looking for a `TOD#` one, but both do use the same rule under the hood. 👍